adagrams/game.py

```diff
@@ -1,11 +1,124 @@
+from random import choice
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
 def draw_letters():
-    pass
+    hand = []
+    while len(hand) < 10: 
+        alphabet, occurence = choice(list(LETTER_POOL.items()))
+        if hand.count(alphabet) < occurence: 
+            hand.append(alphabet)
+    return hand
 
 def uses_available_letters(word, letter_bank):
-    pass
+
+    bank_copy = letter_bank.copy()
+    word = word.upper()
+    for letter in word:
+        if letter not in bank_copy:
+            return False
+        bank_copy.remove(letter)
+    return True
+
 
 def score_word(word):
-    pass
+    score = 0 
+    letter_scores = {
```

Comment:
Consider having this dictionary declared outside of the function, similar to the `LETTER_POOL` dictionary.